apps/token-manager/app/src/screens/Details.js

```diff
@@ -0,0 +1,193 @@
+import React, { useCallback } from 'react'
+import {
+  Accordion,
+  Bar,
+  BackButton,
+  Box,
+  Button,
+  GU,
+  Split,
+  formatTokenAmount,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import SummaryBar from '../components/SummaryBar'
+import SummaryRows from '../components/SummaryRows'
+import VestingInfoBoxes from '../components/VestingInfoBoxes'
+import { useAppLogic, toISODate, useVestedTokensInfo } from '../app-logic'
+import { format, formatDistanceStrict, parseISO } from 'date-fns'
+
+const formatDate = date => `${format(date, 'do MMM yyyy, HH:mm O')}`
+
+function Details({ tokenSymbol, selectHolder, vestings }) {
+  const theme = useTheme()
+  const { selectedHolder } = useAppLogic()
```

Comment:
If we're already using this, perhaps we should just get `selectHolder` here instead of having to pass it down :).